jaxley/modules/base.py

```diff
@@ -41,6 +42,23 @@
 from jaxley.utils.swc import build_radiuses_from_xyzr
 
 
+def only_allow_module(func):
+    """Decorator to only allow the function to be called on Module instances.
```

Comment:
hm maybe this should live in the `utils` folder?

Reply:
I also thought this, but wanted to avoid circular import issues, since `isinstance(self, View)` is used, which means we would have to import `View` in utils. Could work around this by using `self.__class__.__name__ == "View"` but I dislike this a bit. Lemme know your thoughts @michaeldeistler

Reply:
ah, ok...let us keep it where it is then (but maybe move to the bottom of the file?)

Reply:
Also does not work, since it's used in Module and needs to be defined above :/

Reply:
One option would be to fold it into `Module`. Will merge it like it is, feel free to change this if you dislike it.